src/components/Tooltip/Tooltip.js

```diff
@@ -266,6 +266,8 @@ class Tooltip extends Component {
     requestAnimationFrame(() => {
       this.getTriggerPosition();
     });
+
+    document.addEventListener('keydown', this.handleEscKeyPress, false);
```

Comment:
Please clean-up this event listener on un-mount.

Reply:
thanks for the quick review! Keep it up!